source/navitiacommon/navitiacommon/ratelimit.py

```diff
@@ -33,7 +33,7 @@
 import math
 import redis
 import time
-from itertools import izip
+from builtins import zip
```

Comment:
In python 3 `zip()` is the equivalent of `izip()`, but in python 2 they doesn't work the same way, `builtins` form the `futures` package ensure that we always have the desired function accross python version.